$DragonFly: src/gnu/usr.bin/cc3/f771/Attic/com.h.patch,v 1.1 2004/06/21 03:25:33 dillon Exp $
Index: com.h
===================================================================
RCS file: /cvs/src/contrib/gcc-3.3/gcc/f/com.h,v
retrieving revision 1.1
diff -u -r1.1 com.h
--- com.h	2 Feb 2004 23:09:55 -0000	1.1
+++ com.h	21 Jun 2004 02:40:33 -0000
@@ -233,7 +233,7 @@
 void ffecom_finish_progunit (void);
 tree ffecom_get_invented_identifier (const char *pattern, ...)
   ATTRIBUTE_PRINTF_1;
-ffeinfoKindtype ffecom_gfrt_basictype (ffecomGfrt ix);
+ffeinfoBasictype ffecom_gfrt_basictype (ffecomGfrt ix);
 ffeinfoKindtype ffecom_gfrt_kindtype (ffecomGfrt ix);
 void ffecom_init_0 (void);
 void ffecom_init_2 (void);
